index.js

```diff
@@ -306,4 +306,22 @@ Notifications.clearAllNotifications = function() {
 	return this.callNative('clearAllNotifications', arguments)
 }
 
+Notifications.removeAllDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('removeAllDeliveredNotifications')
+  }
+}
+
+Notifications.removeDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('removeDeliveredNotifications', arguments)
+  }
+}
+
+Notifications.getDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('getDeliveredNotifications', arguments)
+  }
+}
+
 module.exports = Notifications;
```

Comment:
@varungupta85 @npomfret like i say in the PR comment, this depends on other PR https://github.com/facebook/react-native/pull/13036 which now is in process of integration, for that reason I put WIP in the title to prevent this was merged before the RN PR... And yes, it was tested